networkx/algorithms/vitality.py

```diff
@@ -14,7 +16,7 @@
 __all__ = ['closeness_vitality']
 
 
-def closeness_vitality(G, node=None, weight=None):
+def closeness_vitality(G, node=None, weight=None, _wiener_index=None):
```

Comment:
Why do we use an underscore here?  It is a public interface and meant to be available to a user.